empress/xscape/plotcostsAnalytic.py

```diff
@@ -3,8 +3,13 @@
 # Plots the cost space using a matplotlib/pyplot
 # Updated June 2020 to plot costs relative to loss rather than to duplication
 
+import sys
 import matplotlib
-matplotlib.use('tkagg') # need this so plt.show() works
+try:
+    matplotlib.use('tkagg') # need this so plt.show() works
+except ImportError:
+    print("Using Agg backend: will not be able to create windows with plots.", file=sys.stderr)
```

Comment:
I think the better description is "Will not be able to create pop-up windows".

Reply:
I think we should also write a comment above this line which says, "tkagg is for pop-up windows while agg cannot create pop-up window. Some environments such as remote servers does not have tkagg. Please refer to #49." What do you think?

Reply:
I added this to `empress/__init__.py`. Once the gui is more fully-supported, this can be moved to the main script, but for now I think it makes sense to allow creating windows from the command-line.